src/index.ts

```diff
@@ -71,16 +70,16 @@ const buildPairsToDeployBundles = flow(
 );
 
 /**
- * buildsToDeployBundles :: Array Codeship.Build.Build -> TaskEither String (Array Deploy.Bundle.Bundle)
+ * buildsToDeployBundles :: Array CircleCI.Build -> TaskEither String (Array Deploy.Bundle.Bundle)
  *
- * Converts the given array of Codeship builds to an array of deploy bundles, wrapped in a
+ * Converts the given array of CircleCI builds to an array of deploy bundles, wrapped in a
  * TaskEither because there are a bunch of async operations that need to happen.
  *
  * We start by getting the current build from the primary Heroku app we want to target.
- * Then, we pair that build with each of the given Codeship builds.
- * Finally, we convert all of the Codeship build / Heroku build pairs into deploy bundles.
+ * Then, we pair that build with each of the given CircleCI builds.
+ * Finally, we convert all of the CircleCI build / Heroku build pairs into deploy bundles.
  */
-const buildsToDeployBundles = (builds: Array<Codeship.Build.Build | CircleCI.Build>) =>
+const buildsToDeployBundles = (builds: Array<CircleCI.Build | CircleCI.Build>) =>
```

Comment:
this looks like an unnecessary union in the type
```suggestion
const buildsToDeployBundles = (builds: Array<CircleCI.Build>) =>
```

Reply:
Thanks!